### PR TITLE
fix notice `Undefined index: operator` #69

### PR DESCRIPTION
**Pull request by Tegos**

### CODE REVIEW

**Comment by oliversarfas** [comment]
Tested as working for Laravel 5.7.14. Please merge in and address the tests as I guess they do not account for "InRaw"

**Comment by mikebronner** [comment]
HI everyone, thanks for submitting this. I will take a look as soon as I can. At the moment I am very busy and can't promise I will get to it right away.

**Comment by drbyte** [comment]
A report for investigating is posted in #195

File: src/CacheKey.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ protected function getQueryColumns(array $columns) : string
 
     protected function getTypeClause($where) : string
     {
-        $type = in_array($where["type"], ["In", "NotIn", "Null", "NotNull", "between", "NotInSub", "InSub", "JsonContains"])
+        $type = in_array($where["type"], ["InRaw", "In", "NotIn", "Null", "NotNull", "between", "NotInSub", "InSub", "JsonContains"])
```
Comment (drbyte) [inline src/CacheKey.php]: I think the other new `types` should be supported here too:
`"InRaw", "NotInRaw", "IntegerInRaw", "IntegerNotInRaw"`

And for completeness probably should be added to the array in `getOtherClauses` too.

Comment (mikebronner) [reply src/CacheKey.php]: Will also need to get the tests written.

**Comment by mikebronner** [comment]
Working on implementing this now.

**Comment by mikebronner** [comment]
@Tegos @oliversarfas @stryaponoff @drbyte Could you guys provide eloquent queries that trigger each of the new types? I have been playing around with it, but so far my queries aren't triggering the new types. (Please reply here in form of comments.)

**Comment by mikebronner** [comment]
@drbyte I wasn't able to find any references to the other types you mentioned above in the Laravel framework code.

Also, I have been able to replicate the triggering the `whereInRaw` query type, and have written a test for it. Update coming shortly.

**Comment by drbyte** [comment]
@mikebronner here are screenshots from the 5.7.13 to 5.7.14 diff I linked to:

screen shot 2018-11-28 at 4 04 44 pm
screen shot 2018-11-28 at 4 04 20 pm
screen shot 2018-11-28 at 4 03 26 pm

**Comment by drbyte** [comment]
@mikebronner BTW thanks for the speedy new release!

**Pull request merged**